client/podman/rest.go

```diff
@@ -194,10 +195,110 @@ func (r *responseReaderID) ReadResponse(response runtime.ClientResponse, consume
 	}
 }
 
-type responseReaderBody struct {
+type responseReaderByteStreamBody struct {
```

Comment:
very minor: MultiplexedBodyReader might be a more descriptive name as it isn't obvious how octet stream is different from byte stream